apacheconfig/loader.py

```diff
@@ -348,6 +368,9 @@ def load(self, filepath, initialize=True):
 
         self._includes.add(filepath)
 
+        if filepath in self._ast_cache:
```

Comment:
This is a great improvement for repetitive loads!

My only concern is that may be we should purge the cache once the top-most `load` is done? Because cached AST may get outdated (of files change) and it takes up memory.

Also, it's best to have unrelated features split up on individual PRs - that keeps clearer history and ease reviews. ;-)

Reply:
Thanks for the Feedback, I'm still getting used to the whole git workflow.  I should have created different branches with each just holding the code changes for the respective feature right?

The top-most `load` is the only one where `initialize` is set to `True` right?

Reply:
Yes! I see you figured that already. ;)